extensions/wikia/DesignSystem/scripts/DesignSystemAuthenticationMenu.js

```diff
@@ -0,0 +1,36 @@
+$(function ($) {
+	'use strict';
+
+	var $signInButton = $('#global-navigation-anon-sign-in'),
+		$registerButton = $('#global-navigation-anon-register');
+
+	function onAuthSuccess() {
+		var redirect = this.url.replace(/.*?redirect=([^&]+).*/, '$1');
```

Comment:
this was olso copyied from current production global nav, should i write test to it?